pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_identifying-module-types.adoc

```diff
@@ -0,0 +1,46 @@
+[id='identifying-module-types_{context}']
+= Identifying {ProductShortName} module types
+:pantheon-module-type: REFERENCE
+
+In Pantheon 2, each module must be identified by type: concept, procedure, or reference.
+
+Module types are described in https://redhat-documentation.github.io/modular-docs/#writing-mod-docs[Types of modules in Red Hat documentation].
+
+Identification is done by using either one or both of the following:
+
+* The correct attribute after the module's title
+* The correct prefix in the module's name
+
+Use of the correct attribute (preferred method)::
+
+One of the following attributes should be entered in the line immediately following the module's title:
+
+* `:pantheon-module-type: CONCEPT`
+* `:pantheon-module-type: PROCEDURE`
+* `:pantheon-module-type: REFERENCE`
+
+====
+.Example
+
+`= Identifying {ProductShortName} module types :pantheon-module-type: REFERENCE`
+
+====
+
+Use of the correct prefix::
+
+Module titles should start with the appropriate choice among the following
```

Comment:
"Module titles" sounds like the title of the topic, but I think this is talking about file names, right? I think that we only use info types in file names, but not in IDs.

Reply:
OK. Changed to "File names of modules... "